### PR TITLE
EZP-28915: Hardcoded Administrator in content edit details section

### PR DESCRIPTION
**Pull request by mikadamczyk**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-28915
| Bug fix?      | yes
| New feature?  |no
| BC breaks?    |no
| Tests pass?   | yes
| Doc needed?   |no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Now Content Item creator name is displayed. It comes from Content Item ownerId field. Displayed date near creator information is changed to contentInfo publishedDate.

screen shot 2018-03-22 at 4 35 30 pm

screen shot 2018-03-23 at 9 12 27 am

#### Checklist:
- [x] Content edit view
- [x] User edit view
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/content/content_edit/content_edit.html.twig [hunk 1]
```diff
@@ -27,7 +27,10 @@
         </h1>
 
         <div class="small">
-            {{ contentType.name }} / {{ 'created_by'|trans({'%name%': 'Administrator'})|desc('Created by %name%') }} / {{ content.versionInfo.creationDate|localizeddate('medium', 'medium', app.request.locale) }} / {{ 'content_id'|trans({'%contentId%': content.id})|desc('Content ID: %contentId%') }}, {{ 'location_id'|trans({'%locationId%': locationId})|desc('Location ID: %locationId%') }}
+            {{ contentType.name }} /
```
Comment (ViniTou) [inline src/bundle/Resources/views/content/content_edit/content_edit.html.twig]: I know I saw this multiple times across latest PR's, are you sure this is the only occurrence that need to be changed?

Comment (lserwatka) [reply src/bundle/Resources/views/content/content_edit/content_edit.html.twig]: There might be something in repository-forms? ping @webhdx

Comment (mikadamczyk) [reply src/bundle/Resources/views/content/content_edit/content_edit.html.twig]: I missed edit_edit view, but I added it already.

Comment (webhdx) [reply src/bundle/Resources/views/content/content_edit/content_edit.html.twig]: It only applies to `content_edit` and `user_edit` which is already done by @mikadamczyk

File: src/lib/RepositoryForms/View/ViewParametersListener.php [hunk 2]
```diff
@@ -39,27 +50,53 @@ public static function getSubscribedEvents()
...
+                ]);
+            }
+
+            $this->processCreator($versionInfo, $contentView);
         }
 
-        /** @var Content $content */
-        $content = $contentView->getParameter('content');
-        $mainLocationId = $content->versionInfo->contentInfo->mainLocationId;
+        if ($contentView instanceof UserUpdateView) {
```
Comment (webhdx) [inline src/lib/RepositoryForms/View/ViewParametersListener.php]: Please create separate methods for User and Content and attach both to the same event name. This makes methods less bloated.

**Comment by lserwatka** [comment]
@mikadamczyk PHPUnit is failing here. There is something wrong with code.

**Pull request merged**